src/app/client/review/write/[id]/_section/WriteReview.tsx

```
@@ -0,0 +1,173 @@
+            />
+          </div>
+        </div>
+      </div>
+
+      <div className='mt-[2.8rem]'>
+        <label className='caption-14-md text-black-10 mb-[1rem] block'>
+          자세한 스냅 촬영 리뷰를 작성해주세요
+        </label>
+        <TextareaField
+          id='review-content'
+          placeholder='리뷰 입력'
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)